app/contracts/user_preferences/params_contract.rb

```diff
@@ -28,11 +28,16 @@
 
 module UserPreferences
   class ParamsContract < ::ParamsContract
+    AVAILABLE_DATE = [nil, 12, 24, 78, 168].freeze
```

Comment:
Not sure what these values represent, according to the specification we have the following values to store:

> - The options for the checkbox in "Start and Finish dates" are:
>   - same day
>   - 1 day before
>   - 3 days before
>   - a week before
> - The options for the checkbox in "When overdue" are:
>   - everyday
>   - every 3 days
>   - every week

Please note, the overdue has different values from the start/due dates.